mmv1/third_party/terraform/services/apigee/resource_apigee_environment_api_revision_deployment_test.go

```diff
@@ -0,0 +1,314 @@
+package apigee_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func TestAccApigeeEnvironmentApiRevisionDeployment_apigeeEnvironmentApiRevisionDeploymentBasicExample(t *testing.T) {
+
+	t.Parallel()
+	context := map[string]interface{}{
+		"org_id":          envvar.GetTestOrgFromEnv(t),
+		"billing_account": envvar.GetTestBillingAccountFromEnv(t),
+		"random_suffix":   acctest.RandString(t, 10),
+	}
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time": {},
+		},
+		CheckDestroy: testAccCheckApigeeEnvironmentApiRevisionDeploymentDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccApigeeEnvironmentApiRevisionDeployment_apigeeEnvironmentApiRevisionDeploymentBasicExample(context),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("google_apigee_environment_api_revision_deployment.deploy", "state"),
+				),
+			},
```

Comment:
```suggestion
			},
			{
				ResourceName:            "google_apigee_environment_api_revision_deployment.deploy",
				ImportState:             true,
				ImportStateVerify:       true,
				ImportStateVerifyIgnore: []string{"override", "sequenced_rollout", "service_account"},
			},
```
Let's also add the import test after the creation step